WPMedia/arc-themes-components
# THEMES-1864: Button styling fix

## Description

#### Jira Ticket: [THEMES-1864](https://arcpublishing.atlassian.net/browse/THEMES-1864)

This PR fixes the styling for the button component. It was out of sync with what is in the Figma style docs. The button styles are located here: https://www.figma.com/design/NBjMF1v4mzwv9jkn7NEq9V/Themes-Block-Styling-2.0?node-id=10862-135976&t=YU12KWjCz73C7KdP-0.

## Test Steps

1. Checkout this branch - `git checkout THEMES-1864`
2. Update dependencies - `npm i`
3. Run Storybook `npm run storybook`
4. Verify that the Button variants (on this page with storybook running http://localhost:6006/?path=/story/components-button--button-variants) `Primary`, `Primary-reverse`, `Secondary`, and `Secondary-reverse` match the Figma docs. Check both the at rest and hover states.


[THEMES-1864]: https://arcpublishing.atlassian.net/browse/THEMES-1864?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
